shims/0.23/src/main/java/org/apache/hadoop/hive/shims/Hadoop23Shims.java

```diff
@@ -1131,7 +1131,7 @@ public void setStoragePolicy(Path path, StoragePolicyValue policy)
       }
     }
     if (needToAddPreserveOption) {
-      params.add("-pbx");
+      params.add("-pb");  //Only Block Size will be preserved.
```

Comment:
We can't remove it in all the cases. for hdfs we should preserve

Reply:
Change done.